### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jlaura**
Draft of the install instructions into the README where many devs / downloaders might end up looking. Used Pandoc for the conversion and then some manual cleanup. I did not catch everything.

## Description

I have taken @jessemapel's open PR and converted the HTML/XML into markdown.
## Motivation and Context

We have docs in a pseudo-opaque place. I would like to make the installation instructions easier for people to find and open PRs to modify.

## How Has This Been Tested?
This has not been tested. This has been previewed in VS Code.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation Change.
## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,370 @@
...
+-   [New ISIS3 environmental variables and their meanings](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#new-environmental-variable-meanings)
+-   [Custom data and test directories](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#custom-data-and-test-data-directories)
+-   [Cleaning builds](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#cleaning-builds)
+-   [Building individual applications/objects](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#building-individual-isis3-applicationsobjects)
+-   [Building ISIS3 documentation](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#building-isis3-documentation)
+-   [What to do if you encounter any problems](https://github.com/USGS-Astrogeology/ISIS3/wiki/Developing-ISIS3-with-cmake#problems)
+
+<span id="RunningOnWindows"></span>
+
+### Running ISIS3 on Windows 10
```
Comment (thareUSGS) [inline README.md]: I feel like, for now, getting ISIS 3.6.0 is a no go for Windows (WSL). I can't get past and opecv install error no matter what I try. Perhaps simply remove until we can figure it out. Or mention the rsync method for ISIS v3.5.2 works for WSL (see blog) and we will update this once we figure it out. 

```
$ cubeatt
cubeatt: error while loading shared libraries: libopencv_core.so.3.2: cannot enable executable stack as shared object requires: Invalid argument
```

Comment (jessemapel) [reply README.md]: :+1:

**Comment by jessemapel** [comment]
Needs an issue?

File: README.md [hunk 2]
```diff
@@ -2,4 +2,370 @@
 
 [![Join the chat at https://gitter.im/USGS-Astrogeology/isis3_cmake](https://badges.gitter.im/USGS-Astrogeology/isis3_cmake.svg)](https://gitter.im/USGS-Astrogeology/isis3_cmake?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Integrated Software for Imagers and Spectrometers ISIS3 public release site
+## Installation
+
+This installation guide is for ISIS3 users interested in installing ISIS3 (3.6.0)+ through conda. If you are a developer, we refer you to our [GitHub repository.](https://github.com/USGS-Astrogeology/ISIS3)
```
Comment (jessemapel) [inline README.md]: If you're a developer please go to the exact same page you're on! We should either add a section about compiling ISIS or link to the wiki page about it.

**Comment by jessemapel** [comment]
I think this is actually #445

**Review by jessemapel — DISMISSED** [review]
I think any changes to the content of the install guide should be done in another PR. There's also some bigger questions about how to structure the README because it should have the build instructions and testing information in it.

The installation guide is primarily user focused and not developer focused.

**Comment by jlaura** [comment]
@thareUSGS @jessemapel I made minor changes to reflect the few comment re: 'developers this page' and 'windows installation'.  If it looks good, feel free to merge and then we can make substantive edits as appropriate.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**